app/models/repo_access.rb

```diff
@@ -19,29 +16,40 @@ class RepoAccess < ActiveRecord::Base
 
   before_validation(on: :create) do
     if organization
-      add_member_to_github_team
-      accept_membership_to_organization
+      add_membership_to_github_organization
+      accept_membership_to_github_organization
```

Comment:
Is there ever a case where we should handle a failing `accept_membership_to_github_organization`?